index.rst

```diff
@@ -302,107 +305,156 @@ Is this case, we could have something like:
+One can see how this can cause confusion when operating the system since "default" can be interpreted in two different ways, e.g.; *schema default* and *operational default*.
 Furthermore, it is usually enough to override a small subset of the *schema defaults* for operations.
 Therefore, to get a full set of applied configurations, users must look at two distinct repositories; the configuration repository (for the modified parameters) and the CSC repository (for the schema defaults).
 
 The proposal to improve this aspect of the system is:
 
-#.  Remove default values from the configuration schema.
+#.  Remove all default values from configuration schema definition in the CSC repository.
 
     - See this :download:`example schema <_static/ATSpectrograph_schema.yaml>` for the ATSpectrograph CSC.
+    - Unit tests will need to utilize configuration files stored in the `tests/data/config` directory as is done for the `ATDome CSC <https://github.com/lsst-ts/ts_ATDome/tree/develop/tests/data/config>`_. See `Salobj documentation <https://ts-salobj.lsst.io>`__) for more details.
+
+#.  In the configuration repository for the given CSC (e.g `ts_config_attcs <https://github.com/lsst-ts/ts_config_attcs>`_ for the ATDome) there shall be a ``_init.yaml`` file defining all values that are expected to be common to all sites and/or be relatively static in operations (we intentionally use "_init" instead of "_default").
+
+    - See this :download:`example _init.yaml <_static/_init.yaml>` for the ATSpectrograph CSC.
+    - This file is the first configuration file loaded by the CSC
+    - It is not possible to load the ``_init.yaml`` file (or any file with a ``_`` prefix) using the ``configurationOverride`` attribute
+    - Note that all CSCs having multiple algorithms, each with different required attributes, must have an initial set of defaults in this file.
+
+#.  Also in the configuration repository for the given CSC, when applicable, there will be a file corresponding to each site where the CSC is used (e.g. ``_summit.yaml, _ncsa.yaml, _base.yaml``).
+    These files contain site specific configuration parameters such as IP addresses and ports.
+    Items in the ``_<site>.yaml`` file will override values that may have been declared in the ``_init.yaml`` file
+    SalObj determines what site file should be loaded automatically by parsing the ``LSST_DDS_PARTITION_PREFIX`` environment variable
 
-#.  On the configuration repository there shall be a ``_base.yaml`` file defining all the base configuration values (we use "base" instead of "default").
+    - See this :download:`example _summit.yaml <_static/_summit.yaml>` for the ATSpectrograph CSC.
+    - This file is the second configuration file to get loaded by the CSC and can override any previously declared values.
+    - It is not possible to load the ``_<site>.yaml`` file (or any file with a ``_`` prefix) using the ``configurationOverride`` attribute
```

Comment:
This duplicates information above (that generated some comment). Is it possible to combine them into one place, or make sure they are both explained exactly the same way?